plantswap/components/Nav/NavItem.js

```diff
@@ -0,0 +1,9 @@
+const NavItem = ({ text, href, active }) => {
+	return (
+		<a href={href} className={`nav__item ${active ? 'active' : ''}`}>
```

Comment:
change <a> tag to `next/link` <Link> tag. <Link> tags do not open a new page but renders te page inside te current view (better user flow).